src/aac/plugins/gen_json/gen_json_impl.py

```diff
@@ -10,20 +10,42 @@
 )
 from aac.validator import validation
 
+
 plugin_name = "gen-json"
 
 
-def print_json(architecture_files: list[str]) -> PluginExecutionResult:
-    """Print the parsed_models from the parsed architecture_files values in JSON format."""
+def print_json(architecture_files: list[str], output_directory: str = None) -> PluginExecutionResult:
+    """Prints the parsed_models from the parsed architecture_files values in JSON format.
+
+       Arguments:
+           architecture_files (list[str]): filepath to the architecture file to convert to JSON.
+           output_directory (str): Directory in which JSON files will be written. (optional)
+
```

Comment:
`flake8` will throw pipeline failures if the spacing is not lined up with the ending `"""`.

```suggestion
    Arguments:
        architecture_files (list[str]): filepath to the architecture file to convert to JSON.
        output_directory (str): Directory in which JSON files will be written. (optional)
```